components/core/UserManager.py

```diff
@@ -51,178 +58,197 @@ def get_user(username):
     db = threadpool.connect()
     if db is not None:
         cursor = db.cursor(MySQLdb.cursors.DictCursor)
-        sql = "SELECT * FROM user WHERE user_name='%s' and blocked=0;" % username
+        sql = "SELECT * FROM user WHERE user_name='%s' and blocked=0;" \
+            % username
         cursor.execute(sql)
         data = cursor.fetchone()
         db.close()
         if data == None:
             return None
         else:
-            return User(data['user_name'], data['password'], data['auth'], data['email'])
+            return User(data['user_name'], data['password'], data['auth'
+                        ], data['email'])
 
 
 def add_user(user):
     db = threadpool.connect()
     if db is not None:
         cursor = db.cursor()
-        sql = "INSERT into user VALUES(%s, MD5(%s), %s, %s, 0, 0);"
+        sql = 'INSERT into user VALUES(%s, MD5(%s), %s, %s, 0, 0);'
         try:
-            cursor.execute(sql, (user.userName, user.password, user.auth, user.email))
+            cursor.execute(sql, (user.userName, user.password,
+                           user.auth, user.email))
             db.commit()
-        except MySQLdb.Error as e:
+        except MySQLdb.Error, e:
             db.rollback()
             return e[1]
-        return "success"
-    return "db connection error"
+        return 'success'
+    return 'db connection error'
+
 
 def add_regular_user(user):
     db = threadpool.connect()
     if db is not None:
         cursor = db.cursor()
-        sql = "INSERT into user VALUES(%s, MD5(%s), %s, %s, 0, 0);"
+        sql = 'INSERT into user VALUES(%s, MD5(%s), %s, %s, 0, 0);'
         try:
-            cursor.execute(sql, (user.userName, user.password, user.auth, user.email))
+            cursor.execute(sql, (user.userName, user.password,
+                           user.auth, user.email))
             db.commit()
-        except MySQLdb.Error as e:
+        except MySQLdb.Error, e:
             db.rollback()
             return e[1]
```

Comment:
someone who read should get an idea what this function returns. make e[1] much meaning full. 
eg : exception[indexof...]

Reply:
Please keep `as ex` syntax instead of `, e` syntax. (It's new syntax and more readable)